config/changed-apps-build.json

```
@@ -53,6 +53,9 @@
       {
         "rootFolder": "check-in",
         "slackGroup": "@check-in-fe"
+      },
+      {
+        "entryName": "lgy"
```

```suggestion
        "rootFolder": "lgy"
```
Since this is a grouped app